pkg/status/status.go

```diff
@@ -124,15 +125,13 @@ func GetAndFormatStatus() ([]byte, error) {
 
 // GetCheckStatusJSON gets the status of a single check as JSON
 func GetCheckStatusJSON(c check.Check, cs *checkstats.Stats) ([]byte, error) {
-	s, err := GetStatus(false)
-	if err != nil {
-		return nil, err
-	}
-	checks := s["runnerStats"].(map[string]interface{})["Checks"].(map[string]interface{})
+	stats := make(map[string]interface{})
+	stats = getRunnerStats(stats)
```

Comment:
Can you explain why you call `getRunnerStats` instead of `GetStatus(false)`?

Reply:
Actually, on one of the PR I have that extracts the `render` functions to a separate status subpackage, we can get rid of `GetCheckStatusJSON` and `GetCheckStatus` 

https://github.com/DataDog/datadog-agent/pull/20776/files#diff-4a83275fbbf7764c4d251dde368ca784c146ed15c8b49dbc9946ecb0450e57b2R368-R459

https://github.com/DataDog/datadog-agent/pull/20776/files#diff-1deea3647518183ee4915dd3c10ccfc7bb60e1ed860673c99eefaf5cfeefaa6bL127-L143